news-bundle/src/EventListener/DataContainer/LayoutFeedOptionsListener.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\NewsBundle\EventListener\DataContainer;
+
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\CoreBundle\ServiceAnnotation\Callback;
+use Contao\NewsBundle\Controller\Page\NewsFeedController;
+use Contao\PageModel;
+
+/**
+ * @Callback(table="tl_layout", target="fields.newsfeeds.options")
+ */
```

Comment:
This should be a PHP 8 attribute as well.

Reply:
Fixed in d8ddd2965